integration_tests/mockApis/auth.ts

```diff
@@ -122,7 +122,7 @@ const stubUser = (name: string) =>
   stubFor({
     request: {
       method: 'GET',
-      urlPattern: '/auth/api/user/me',
+      urlPattern: '/users/me',
```

Comment:
[This](https://github.com/ministryofjustice/hmpps-template-typescript/pull/247/files#r1371550755) means that when stubbing you need to prefix these with `manage-users-api`

Reply:
And in fact because we've got separate files per API, might make sense pulling these stubs out into there own `mockApis/manageUsersApi.ts` file